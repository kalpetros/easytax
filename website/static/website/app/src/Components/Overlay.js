import styles from '../../../css/src/Components/Overlay.css';

import React from 'react';

class Overlay extends React.Component {
    constructor(props) {
        super(props);
        this.state = {
        };

    }
    componentDidMount() {
    }
    render() {
	return(
            <div className={styles.overlay}>
              {this.props.children}
	    </div>
	);
    }
}

export{Overlay};
