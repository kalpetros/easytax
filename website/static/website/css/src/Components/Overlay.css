.overlay {
    background: linear-gradient(90deg,
				rgba(255, 255, 255, 0.5),
				rgba(15, 17, 37, 0.9));
    position: fixed;
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;
    z-index: 9999;
}
