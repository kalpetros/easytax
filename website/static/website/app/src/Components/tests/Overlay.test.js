import {Overlay} from '../Overlay';

import renderer from 'react-test-renderer';

import React from 'react';

test('Renders an overlay', () => {
  const component = renderer.create(
    <Overlay>Test</Overlay>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
