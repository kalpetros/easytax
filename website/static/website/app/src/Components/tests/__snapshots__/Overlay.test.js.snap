// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders an overlay 1`] = `
<div
  className="overlay"
>
  Test
</div>
`;
